.build/package.json

```diff
@@ -55,6 +56,7 @@
     "bundle-profile:analyze": "npm run build:bundle-profile && webpack-bundle-analyzer ./stats.json",
     "start": "webpack-cli serve",
     "start:dev": "webpack serve --hot --color --progress --config webpack.dev.js",
+    "start:prod": "serve dist -l 9001 --cors",
```

Comment:
to test prod build in the context of the `application-services-ui` container, can first run `npm run build` followed by `npm run start:prod`